# Single-RNA-level analysis of full-length HIV-1 RNAs reveals functional redundancy of m6As
Alice Baek<sup>1,2,3#</sup>, Ga-Eun Lee<sup>1,2,3,4#</sup>, Sarah Golconda<sup>1,2,3</sup>, Asif Rayhan<sup>5</sup>, Anastasios A. Manganaris<sup>4,6</sup>, Shuliang Chen<sup>1,2</sup>, Nagaraja Tirumuru<sup>1,2</sup>, Hannah Yu<sup>1,2,3</sup>, Shihyoung Kim<sup>1,2,3</sup>, Christopher Kimmel<sup>2,4</sup>, Olivier Zablocki<sup>7,8</sup>, Matthew B Sullivan<sup>7,8,9</sup>, Balasubrahmanyam Addepalli<sup>5</sup>, Li Wu<sup>10</sup>, and Sanggu Kim<sup>1,2,3,4*</sup>

<sup>1</sup>Center for Retrovirus Research, The Ohio State University, Columbus, OH, USA

<sup>2</sup>Department of Veterinary Biosciences, The Ohio State University, Columbus, OH, USA

<sup>3</sup>Infectious Diseases Institute, The Ohio State University, Columbus, OH, USA

<sup>4</sup>Translational Data Analytics Institute, The Ohio State University, Columbus, OH, USA

<sup>5</sup>Rieveschl laboratories for mass spectrometry, Department of Chemistry, University of Cincinnati, Cincinnati, OH, USA

<sup>6</sup>Department of Computer Science and Engineering, The Ohio State University, Columbus, OH, USA

<sup>7</sup>Center of Microbiome Science, Ohio State University, Columbus, OH, USA

<sup>8</sup>Department of Microbiology, Ohio State University, Columbus, OH, USA

<sup>9</sup>Department of Civil, Environmental and Geodetic Engineering, Ohio State University, Columbus, OH, USA

<sup>10</sup>Department of Microbiology and Immunology, Carver College of Medicine, University of Iowa, Iowa City, IA, USA.

<sup>#</sup>These authors contributed equally: Alice Baek, Ga-Eun Lee

<sup>*</sup>Corresponding author: Sanggu Kim,✉e-mail:kim.6477@osu.edu

**SUMMARY**

HIV-1 exploits every aspect of RNA, a versatile macromolecule that undergoes various post-transcriptional modifications, to maximize its replication. Although the importance of chemical modifications on RNA has been recognized, their evolutionary benefits and precise roles in HIV-1 replication remain unclear. Most studies have provided only population-averaged values of modifications for fragmented RNAs at low resolution and have relied on indirect analyses of the phenotypic effects of perturbing host effectors, neglecting site-specificity and intra-RNA heterogeneity1-9. Here, we developed a new RNA-library-preparation method for full-length direct RNA sequencing and analyzed HIV-1-specific modifications at the single-RNA level. Our analysis revealed that the HIV-1 modification landscape is unexpectedly simple, showing three predominant N6-methyladenosine (m6A) modifications near the 3' end. More densely installed in viral mRNAs than in genomic RNAs, these m6As play a crucial role in maintaining normal levels of RNA splicing and translation. We also discovered that HIV-1 generates diverse RNA subspecies with distinct ensembles of the m6As and that these m6As regulate splicing independently of each other. Our single-RNA-level study demonstrates that HIV-1 tolerates functionally redundant m6As to provide stability and resilience to viral replication while minimizing the risk of unpredictable mutagenesis – a novel RNA-level strategy similar to bet-hedging in evolutionary biology.

**USAGE**

The complete methodology for training a model is described in the manuscript (see Supplementary Note 2).

The pretrained model can be executed by following the instructions in the USAGE.org file. See (USAGE.org).

A script for training is also provided.
